nemo_skills/dataset/librispeechmix-sot/prepare.py

```
@@ -0,0 +1,410 @@
+    """Load a JSONL file."""
+    rows = []
+    with open(path, "rt", encoding="utf-8") as fin:
+        for line in fin:
+            if line.strip():
+                rows.append(json.loads(line))
+    return rows
+
+
+def mixed_duration(row: dict) -> float:
+    """Return mixed utterance duration from source delays and durations."""
+    return float(max(delay + duration for delay, duration in zip(row["delays"], row["durations"])))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)